xarray/tests/test_plot.py

```diff
@@ -1806,7 +1806,6 @@ def test_colors_np_levels(self) -> None:
         cmap = artist.cmap
         assert isinstance(cmap, mpl.colors.ListedColormap)
         colors = cmap.colors
-        assert isinstance(colors, list)
```

Comment:
That's now an numpy array. I am reasonably sure that is a behaviour change in matplotlib and I don't think we should try to preserve that.

https://github.com/matplotlib/matplotlib/blame/84464dd085210fb57cc2419f0d4c0235391d97e6/lib/matplotlib/contour.py#L190

Reply:
I think the instance check was only there to make mypy happy. We don't really care what type colors is, as long as it is iterable.